tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -2884,50 +2885,207 @@ TEST_F(CompilerTest, CompileWhenIncludeThenLoadUsed) {
                         pInclude->GetAllFileNames().c_str());
 }
 
-TEST_F(CompilerTest, CompileWhenAllIncludeCombinations) {
-  static auto NormalizeForPlatform = [](const std::wstring &s) -> std::wstring {
+static std::wstring NormalizeForPlatform(const std::wstring &s) {
 #ifdef _WIN32
-    wchar_t From = L'/';
-    wchar_t To = L'\\';
+  wchar_t From = L'/';
+  wchar_t To = L'\\';
 #else
-    wchar_t From = L'\\';
-    wchar_t To = L'/';
+  wchar_t From = L'\\';
+  wchar_t To = L'/';
 #endif
-    std::wstring ret = s;
-    for (wchar_t &c : ret) {
-      if (c == From)
-        c = To;
-    }
-    return ret;
-  };
+  std::wstring ret = s;
+  for (wchar_t &c : ret) {
+    if (c == From)
+      c = To;
+  }
+  return ret;
+};
 
-  class SimpleIncludeHanlder : public IDxcIncludeHandler {
```

Comment:
Question: Can you confirm that the only change to this was indentation?

Reply:
Actually there was a call to `NormalizeForPlatform` that was removed. It would have actually defeated the purpose of the test during the recompile.